'Element' is imported from both 'package://...' and 'package://...'

I am trying to resolve this issue below.
But I can't find the files originated in the error: "page.dart" and "navigator.dart" does anyone have an idea of what may be throwing this error?

Config:
Windows 10 Pro
Flutter Config:
```
Flutter 1.20.2 • channel stable • https://github.com/flutter/flutter.git
Framework • revision bbfbf1770c (3 weeks ago) • 2020-08-13 08:33:09 -0700
Engine • revision 9d5b21729f
Tools • Dart 2.9.1
```

Copy & paste:
```
../../AppData/Local/Pub/Cache/hosted/pub.dartlang.org/intro_views_flutter-2.8.0/lib/intro_views_flutter.dart:241:11: Error: 'Page' is imported from both 'package:flutter/src/widgets/navigator.dart' and 'package:intro_views_flutter/UI/page.dart'.
          Page(
          ^^^^
../../AppData/Local/Pub/Cache/hosted/pub.dartlang.org/intro_views_flutter-2.8.0/lib/intro_views_flutter.dart:249:20: Error: 'Page' is imported from both 'package:flutter/src/widgets/navigator.dart' and 'package:intro_views_flutter/UI/page.dart'.
            child: Page(
                   ^^^^


FAILURE: Build failed with an exception.

* Where:
Script 'C:\flutter\packages\flutter_tools\gradle\flutter.gradle' line: 896

* What went wrong:
Execution failed for task ':app:compileFlutterBuildDebug'.
> Process 'command 'C:\flutter\bin\flutter.bat'' finished with non-zero exit value 1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 17s
Exception: Gradle task assembleDebug failed with exit code 1
Exited (sigterm)
```

![Captura de Tela (30)](https://user-images.githubusercontent.com/51419598/92022822-c4520700-ed53-11ea-84f7-d6f754f4e53c.png)

--- a/pubspec.yaml
+++ b/pubspec.yaml
@@ -33,7 +33,7 @@ dependencies:
   flutter_share_me: ^0.9.1
   flutter_spinkit: ^4.1.2
   http: ^0.12.0+4
-  intro_views_flutter: ^2.8.0
+  intro_views_flutter: ^2.8.1
   launch_review: ^2.0.0
   material_design_icons_flutter: ^4.0.5045
   random_string: ^2.0.1

